### PR TITLE
update compute status display text

### LINKED ISSUE #1562: Apps: Update status text
Update the relevant compute statuses display text to
- "Not started"
- "In progress. You may return later to use results in the visualization."
- "Complete, results saved in the system"

### PR DESCRIPTION
**Pull request by asizemore**
Resolves #1562 

Adds a new map between internal compute statuses and user-friendly statuses.

### CODE REVIEW

**Comment by asizemore** [comment]
A note to reviewers - hopefully should be a quick review and only needs one review. Tagged you both just to save time if one of you is able to get to it sooner!

**Review by dmfalke — CHANGES_REQUESTED** [review]
I just have one small refinement to the type definition of `jobStatusDisplay`.

File: src/lib/core/components/computations/RunComputeButton.tsx [hunk 1]
```diff
@@ -53,14 +53,26 @@ const colorMap: Record<JobStatus, string> = {
...
 };
 
+// Replace internal job status with user-friendly status messages
+const jobStatusDisplay: Record<JobStatus, string> = {
+  'no-such-job': 'Not started.',
+  requesting: 'Requesting.',
+  queued: 'Queued.',
+  'in-progress':
+    'In progress. You may return later to use results in the visualization.',
+  complete: 'Complete, results saved in the system.',
```
Comment (jernestmyers) [inline src/lib/core/components/computations/RunComputeButton.tsx:63]: I'd probably be consistent here and make this two sentences: `'Complete. Results saved in the system.'`

Just a thought. Not necessary at this stage though.

**Review by jernestmyers — APPROVED** [review]

File: src/lib/core/components/computations/RunComputeButton.tsx (hunk 1 above)
Comment (asizemore) [reply src/lib/core/components/computations/RunComputeButton.tsx]: good catch!

**Review by dmfalke — APPROVED** [review]

File: src/lib/core/components/computations/RunComputeButton.tsx [hunk 2]
```diff
@@ -53,14 +53,26 @@ const colorMap: Record<JobStatus, string> = {
...
+  expired: 'Results expired.',
+  failed: 'Failed. Contact the VEuPathDB team for support.',
+};
```
[the remark this answers is not in the document]
Comment (asizemore) [reply src/lib/core/components/computations/RunComputeButton.tsx]: Oh that's cool! Thank you!

**Pull request merged**